README.md

```diff
@@ -1,2 +1,352 @@
+
+
 # Container
-Container in go
+## A subtle deblurring of the concepts
+
+### What is a Container?
+
+Containers are structured files a.k.a. file system layers and configuration files as JSON blobs to resulting with a packaging scheme and runtime isolation. This is a lot to take in. Let us divide and conquer this.
+
+![file system meme...](/images/fs_meme.png)
+---
+**What does structured files or a file system mean?**
+
+![file system meme...](/images/fs_meme_2.jpeg)
+
+
+Information is contained within a file. A directory itself is a file too. The only difference is that it can contain other files/directories. Directories have children in a tabular form while their content is null. Thus their size is 0.
+
+if we `mdls images/` in this repository, we will see the following. `mdls` stands for Metadata List and as its name suggests it is for listing the metadata of the specified **file**.
+
+Metadata of `images/`             |  metadata of `README.md`
+:-------------------------:|:-------------------------:
+![metadata of folder](/images/metadata_of_images.png)  | ![metadata of README.md file](/images/metadata_of_md.png)
+
+
+You can see that they are not awfully different. It is trivial to see know that a directory is a file itself that can comtain multiple other files.
+
+
+
+
+We can say that containers are a collection of linux technologies that run one or more linux process(es). That is why we will discover 'structured files / file systems' in Linux itself. 
+
+If I recklessly type `tree /` (
+`/` *refers to the root directory. Root directory is - as its name suggests - the directory from which all the other directories branches off.* 
+), my life will flash before my eyes. 
+
+![Without limited tree cmd](/images/tree_without_limits.mov)
+
+We will not cover all of this obviously ... Let us limit the `tree` command by telling it to show only the 1st children of the root directory by `tree -L 1 /`.
+
+![First Degree Childs of the root directory](/images/from_the_root_of_the_ubuntu_fs.png)
+
+---
+**`/bin`** includes the binaries - programs that you can run - which are essential user commands. `cat`, `mv`, `kill`, `ls`, `rm` and so on all of them are within **`/bin`**. There can be other **`/bin`** directories within the file system.
+
+You might be wondering why there is an arrow `->` after **`/bin`**. `tree` shows symbolic links in that format : `name -> real-path`. A symbolic link - you may have encountered it referred as a symlink, shell link or a soft link - points to another file or directory on the system or a connected file system. A fancy way of saying a `shortcut`. So my **`/bin`** directory actually points to **`usr/bin`** and the binaries that 'I' - as a user - use and run are in that folder.
+
+You see yourself where that programs or commands reside with `which`. For example, the command to move stuff around `mv`:
```

Comment:
> You see yourself

did you mean -> *you see for yourself* ?
if so, you can simply get rid of `yourself` as well. Does not add much value to the content imo

Reply:
I don't know even remember what I meant ... Not this one for sure.